runtime/testnet/src/config/governance.rs

```diff
@@ -0,0 +1,68 @@
+use frame_support::{
+	parameter_types,
+	traits::{EitherOfDiverse, NeverEnsureOrigin},
+};
+use frame_system::EnsureRoot;
+use pallet_collective::EnsureProportionAtLeast;
+use sp_core::crypto::Ss58Codec;
+
+use crate::{
+	config::system::RuntimeBlockWeights, AccountId, BlockNumber, Runtime, RuntimeCall,
+	RuntimeEvent, RuntimeOrigin, Weight, DAYS,
+};
+
+// Type aliases for council origins.
+type AtLeastThreeFourthsOfCouncil = EitherOfDiverse<
+	EnsureRoot<AccountId>,
+	EnsureProportionAtLeast<AccountId, CouncilCollective, 3, 4>,
+>;
+type UnanimousCouncilVote = EitherOfDiverse<
+	EnsureRoot<AccountId>,
+	EnsureProportionAtLeast<AccountId, CouncilCollective, 1, 1>,
+>;
+
+/// SUDO account set at genesis.
+const SUDO_ACCOUNT: &str = "5FPL3ZLqUk6MyBoZrQZ1Co29WAteX6T6N68TZ6jitHvhpyuD";
```

Comment:
Just notes for other reviewers: this is the correct Sudo address as exists on Testnet currently.

![Screenshot 2025-03-03 at 11 20 37 AM](https://github.com/user-attachments/assets/ff4ca7cb-a5f4-41f8-babc-1d183eb12edc)
![Screenshot 2025-03-03 at 11 20 46 AM](https://github.com/user-attachments/assets/01023d83-8405-402a-be21-ce5f921bb151)